rules/falco_rules.yaml

```diff
@@ -1910,11 +1913,8 @@
 # doesn't allow exclusions of a full pattern, only single characters.
 - macro: sensitive_mount
   condition: (container.mount.dest[/proc*] != "N/A" or
-              container.mount.dest[/var/run/docker.sock] != "N/A" or
-              container.mount.dest[/var/run/crio/crio.sock] != "N/A" or
               container.mount.dest[/run/containerd/containerd.sock] != "N/A" or
-              container.mount.dest[/var/lib/kubelet] != "N/A" or
-              container.mount.dest[/var/lib/kubelet/pki] != "N/A" or
+              container.mount.dest[/var*] != "N/A" or
```

Comment:
can you please test with scenario of pod using service account? I bet it will trigger the alert with your changes?

Reply:
Hey @Kaizhe 

are there any other blockers for you? I would like to include this PR in 0.31.0. wdyt?

Reply:
My concern is this is going to trigger a lot of false positives.

Reply:
Hey @sberkovich 

as discussed privately, this solution :point_right: https://github.com/falcosecurity/falco/pull/1810/commits/c28fd3817c628378c017ad2e174aabae287dbd40#diff-ea7e90b2fb13b5aa2c8e6799c1ea20612e8bd29a13c61dfb3653f03f0235436eR1926 seems interesting. 

I'm not aware if there are other legit exceptions, but the two included in `allowed_mount` seem ok to me.
@Kaizhe wdyt?

Reply:
It would be hard to iterate all the scenarios like secret, configmap etc. and /var directory is supposed to mean variant when Unix/Linux introduce it.